### ISSUE TITLE
do we need to expose port 1680 externally?

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/helium-miner-software#125]
- [x] I understand that this is only to report software bugs and **not** for customer support. Please use [nebra.io/helium-support](https://nebra.io/helium-support).

**Describe the bug**

Currently in the [docker-compose](https://github.com/NebraLtd/helium-miner-software/blob/master/docker-compose.yml) we have for the miner container:
```
    ports:
      - "44158:44158/tcp"
      - "1680:1680/udp"
```
Whilst 44158 needs to be exposed externally, 1680 is only used for communication with the packet forwarder and so I think we could just `expose` it instead.

**Comment by shawaj** [comment]
Tested in light gateway and works fine https://github.com/NebraLtd/light-hotspot-software/commit/775a86871ca51489cf57395d7ceba8d81d6510ef#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3

**Comment by shawaj** [comment]
Closed by #140
